### PR TITLE
Build release package

### PR DESCRIPTION
**Pull request by wenwei-dev**
The dependencies may not accurate because some of them may be the dependencies of opencog. Should be verified by people who familiars with opencog.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -551,6 +553,12 @@ SET(CPACK_SOURCE_IGNORE_FILES
 	"^${PROJECT_BINARY_DIR}/examples/server/derivedserver$"
 	"^${PROJECT_BINARY_DIR}/examples/hopfield/hopfield$"
 )
+
+SET(CPACK_PACKAGING_INSTALL_PREFIX "/usr/local")
+SET(CPACK_DEBIAN_PACKAGE_DEPENDS
+    "rlwrap (>= 0.37), guile-2.0 (>= 2.0.9), bzip2 (>= 1.0.6), cython (>= 0.20.1), python (>= 2.7.5), python-zmq (>= 14.0.1), python-simplejson (>= 3.3.1), libzmq3 (>= 4.0.4), libboost-date-time1.54.0, libboost-filesystem1.54.0, libboost-program-options1.54.0, libboost-regex1.54.0, libboost-serialization1.54.0, libboost-system1.54.0, libboost-thread1.54.0, binutils (>= 2.24), uuid (>= 1.6.2), protobuf-compiler (>= 2.5.0), libsdl-gfx1.2-4 (>= 2.0.23), libssl1.0.0 (>= 1.0.1), tcl (>= 8.6.0), tcsh (>= 6.18.01), libfreetype6 (>= 2.5.2), libatlas-base-dev, gfortran (>= 4:4.8.2), unixodbc (>= 2.2.14p2), odbc-postgresql (>= 1:09.02.0100), postgresql-client (>= 9.3), cogutils (>= 0.1.4)"
```
Comment (linas) [inline CMakeLists.txt]: I'm pretty sure sdl-gfx is not needed (that's a graphics/drawing library) also freetype should not be needed. or tcl to tcsh ...

File: CMakeLists.txt [hunk 2]
```diff
@@ -498,13 +498,14 @@ ADD_CUSTOM_TARGET(cscope
...
 # packaging
 SET(CPACK_PACKAGE_DESCRIPTION_SUMMARY "The Open Cognition Framework")
-SET(CPACK_PACKAGE_NAME "opencog")
+SET(CPACK_PACKAGE_NAME "atomspace")
 SET(CPACK_PACKAGE_VENDOR "opencog.org")
 SET(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_CURRENT_SOURCE_DIR}/README.md")
 SET(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/LICENSE")
 SET(CPACK_PACKAGE_VERSION_MAJOR "0")
 SET(CPACK_PACKAGE_VERSION_MINOR "1")
 SET(CPACK_PACKAGE_VERSION_PATCH "4")
```
Comment (linas) [inline CMakeLists.txt]: we should probably set version major to 1 by now.

File: CMakeLists.txt (hunk 1 above)
Comment (AmeBel) [inline CMakeLists.txt]: rlwrap is a tool, so not sure it should be included

File: CMakeLists.txt [hunk 3]
```diff
@@ -551,6 +563,12 @@ SET(CPACK_SOURCE_IGNORE_FILES
 	"^${PROJECT_BINARY_DIR}/examples/server/derivedserver$"
 	"^${PROJECT_BINARY_DIR}/examples/hopfield/hopfield$"
 )
+
+SET(CPACK_PACKAGING_INSTALL_PREFIX "/usr/local")
+SET(CPACK_DEBIAN_PACKAGE_DEPENDS
+    "rlwrap (>= 0.37), guile-2.0 (>= 2.0.9), bzip2 (>= 1.0.6), cython (>= 0.20.1), python (>= 2.7.5), python-zmq (>= 14.0.1), python-simplejson (>= 3.3.1), libzmq3 (>= 4.0.4), libboost-date-time1.54.0, libboost-filesystem1.54.0, libboost-program-options1.54.0, libboost-regex1.54.0, libboost-serialization1.54.0, libboost-system1.54.0, libboost-thread1.54.0, binutils (>= 2.24), uuid (>= 1.6.2), protobuf-compiler (>= 2.5.0), libssl1.0.0 (>= 1.0.1), libatlas-base-dev, gfortran (>= 4:4.8.2), unixodbc (>= 2.2.14p2), odbc-postgresql (>= 1:09.02.0100), postgresql-client (>= 9.3), cogutils (>= 0.1.4)"
+)
```
Comment (linas) [inline CMakeLists.txt]: want cogutils >= 2.0.2 since that will be the current version of cogutils.

File: CMakeLists.txt [hunk 4]
```diff
@@ -513,9 +524,10 @@ SET(CPACK_SOURCE_STGZ "OFF")
 SET(CPACK_SOURCE_TBZ2 "ON")
 SET(CPACK_SOURCE_TGZ "OFF")
 SET(CPACK_SOURCE_TZ "OFF")
+SET(CPACK_PACKAGE_CONTACT "team@opencog.org")
```
Comment (linas) [inline CMakeLists.txt]: That mailing list doesn't exist (or is unused, unmaintained) Use <opencog@googlegroups.com> instead.

File: CMakeLists.txt [hunk 5]
```diff
@@ -551,6 +563,12 @@ SET(CPACK_SOURCE_IGNORE_FILES
 	"^${PROJECT_BINARY_DIR}/examples/server/derivedserver$"
 	"^${PROJECT_BINARY_DIR}/examples/hopfield/hopfield$"
 )
+
+SET(CPACK_PACKAGING_INSTALL_PREFIX "/usr/local")
+SET(CPACK_DEBIAN_PACKAGE_DEPENDS
+    "rlwrap (>= 0.37), guile-2.0 (>= 2.0.9), bzip2 (>= 1.0.6), cython (>= 0.20.1), python (>= 2.7.5), python-zmq (>= 14.0.1), python-simplejson (>= 3.3.1), libzmq3 (>= 4.0.4), libboost-date-time1.54.0, libboost-filesystem1.54.0, libboost-program-options1.54.0, libboost-regex1.54.0, libboost-serialization1.54.0, libboost-system1.54.0, libboost-thread1.54.0, binutils (>= 2.24), uuid (>= 1.6.2), protobuf-compiler (>= 2.5.0), libssl1.0.0 (>= 1.0.1), libatlas-base-dev, gfortran (>= 4:4.8.2), unixodbc (>= 2.2.14p2), odbc-postgresql (>= 1:09.02.0100), postgresql-client (>= 9.3), cogutils (>= 0.1.4)"
```
Comment (linas) [inline CMakeLists.txt]: I don't beleive that rlwrap and bzip2 are needed to build, install or run the atomspace.

Comment (linas) [inline CMakeLists.txt]: Also, I don't think that binutils is needed.

Comment (linas) [inline CMakeLists.txt]: Also, I don't think postgresql-client or gfortran or cython are needed, not for the run-time. These may be needed to build the atomspace, but building the atomspace is different from just running it.  Also, I'm thinking that python-simplejson and python-zmq

Comment (linas) [inline CMakeLists.txt]: And I don't think that uuid is needed either for the build, or for the install.

**Pull request merged**